srcpkgs/inkscape/template

```diff
@@ -1,27 +1,27 @@
 # Template file for 'inkscape'
 pkgname=inkscape
-version=1.0.2
-revision=3
-wrksrc="inkscape-${version}_2021-01-15_e86c870879"
+version=1.1
+revision=1
+wrksrc="${pkgname}-${version}_2021-05-24_c4e8f9ed74"
 build_style=cmake
 # builds executables then runs checks
 # FIXME: some tests still fail on musl
 make_check_target=check
 hostmakedepends="automake pkg-config libtool intltool gettext
- glib-devel perl-XML-Parser tar which xz"
-makedepends="harfbuzz-devel libsoup-devel gsl-devel pango-devel
+ glib-devel perl-XML-Parser which"
+makedepends="harfbuzz-devel libsoup-devel gsl-devel pango-devel libatomic_ops-devel
  double-conversion-devel gc-devel libwpd-devel libcdr-devel libvisio-devel
  poppler-devel libwpd-devel potrace-devel gtkmm-devel gdl-devel gtkspell3-devel
  aspell-devel libxslt-devel libgomp-devel libwpg-devel poppler-glib-devel"
 depends="desktop-file-utils hicolor-icon-theme python3-lxml python3-numpy python3-scour"
-checkdepends="$depends gtest-devel ImageMagick cantarell-fonts"
+checkdepends="$depends gtest-devel ImageMagick cantarell-fonts noto-fonts-cjk-20190416_2
```

Comment:
Why are you specifying the version for `noto-fonts-cjk`? That will break whenever it's updated.

Reply:
I only used the package that I found for the font. Can I use `noto-fonts-cjk` instead?

Reply:
Yes.

Reply:
ah, that works. Thanks!